bootstrap/events/producer/streams_test.go

```diff
@@ -241,17 +270,22 @@ func TestUpdate(t *testing.T) {
 	assert.Nil(t, err, fmt.Sprintf("got unexpected error: %s", err))
 
 	cases := []struct {
-		desc   string
-		config bootstrap.Config
-		token  string
-		err    error
-		event  map[string]interface{}
+		desc         string
+		config       bootstrap.Config
+		token        string
+		authResponse *magistrala.AuthorizeRes
+		authorizeErr error
+		identifyErr  error
+		updateErr    error
+		err          error
+		event        map[string]interface{}
```

Comment:
Add the following cases this applies to other places too
- update with failed to identify a user
- update with failed to authorize user
- update with failed to fetch thing
- update with failed to fetch channel
- update with failed to fetch config
- update with failed to list existing
- update with failed to update connections